client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -294,3 +294,20 @@ $width-left-panel-card: 55%;
 	}
 }
 
+.email-provider-stacked-features__logos {
+	column-gap: 3px;
+	display: flex;
+	flex-wrap: wrap;
+	margin-left: 30px;
+	margin-top: 0;
+	row-gap: 5px;
+	@include break-xlarge {
+		justify-content: space-between;
+		row-gap: 3px;
```

Comment:
I'm not sure why we need to define a different (and smaller) gap on wide screens.

Reply:
I fine tuned it to 3px because at 4px the logos were occupying 2 rows.

Reply:
I'm not sure to understand, doesn't `row-gap` define the space between rows (and not between columns)? I don't see how that would make logos wrap on two lines.

Reply:
You are right, I confused it!